apps/frontend/src/libs/components/sidebar/sidebar-items.tsx

```diff
@@ -0,0 +1,28 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+
+import { type NavigationItem } from "./sidebar-item/type.js";
+
+const sidebarItems: NavigationItem[] = [
```

Comment:
```suggestion
const NAVIGATION_ITEMS: NavigationItem[] = [
```
Also this file should be in `src/libs/constants/navigation-items.constants.ts` and  NavigationItem type in `src/libs/types/navigation-item.type.ts`